File: Modul-4/PRAK404/PRAK404-2310817210010-Harry-Pratama-Yunus.c
#include <stdio.h>

int main(){
    while(1){
        printf("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit\n");
        float a,b,c,d;
        printf("Masukkan Pilihan : ");
        scanf("%f",&a);
        if(a==1){
        printf("Masukkan Nilai Pertama : ");
        scanf("%f",&b);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f",&c);
        d=b+c;
        printf("Hasil pertambahan antara %.2f dengan %.2f adalah %.2f\n",b,c,d);
        continue;
        }
        else if(a==2){
        printf("Masukkan Nilai Pertama : ");
        scanf("%f",&b);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f",&c);
        d=b-c;
        printf("Hasil pengurangan antara %.2f dengan %.2f adalah %.2f\n",b,c,d);
        continue;
        }
        else if(a==3){
        printf("Masukkan Nilai Pertama : ");
        scanf("%f",&b);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f",&c);
        d=b*c;
        printf("Hasil perkalian antara %.2f dengan %.2f adalah %.2f\n",b,c,d);
        continue;
        }
        else if(a==4){
        printf("Masukkan Nilai Pertama : ");
        scanf("%f",&b);
        printf("Masukkan Nilai Kedua : ");
        scanf("%f",&c);
        d=b/c;
        printf("Hasil pembagian antara %.2f dengan %.2f adalah %.2f\n",b,c,d);
        continue;
        }
        else if(a==5){
        printf("Terimakasih, telah menggunakan kalkulator Harry Pratama Yunus");
        break;
        }
        else{
        printf("Input anda salah, silahkan coba lagi\n");
        continue;
        }
    }
return 0;
}

File: Modul-4/PRAK404/PRAK404-2310817210010-Harry-Pratama-Yunus.py
while True :
    print("Pilih program\n1. Penjumlahan\n2. Pengurangan\n3. Perkalian\n4. Pembagian\n5. Exit")
    print("Masukkan Pilihan : ",end="")
    a=float(input())
    if a==1:
        print("Masukkan Nilai Pertama : ",end="")
        b=float(input())
        print("Masukkan Nilai Kedua : ",end="")
        c=float(input())
        d=b+c
        print("Hasil pertambahan antara {:.2f} dengan {:.2f} adalah {:.2f}".format(b,c,d))
        continue
    elif a==2:
        print("Masukkan Nilai Pertama : ",end="")
        b=float(input())
        print("Masukkan Nilai Kedua : ",end="")
        c=float(input())
        d=b-c
        print("Hasil pengurangan antara {:.2f} dengan {:.2f} adalah {:.2f}".format(b,c,d))
        continue
    elif a==3:
        print("Masukkan Nilai Pertama : ",end="")
        b=float(input())
        print("Masukkan Nilai Kedua : ",end="")
        c=float(input())
        d=b*c
        print("Hasil perkalian antara {:.2f} dengan {:.2f} adalah {:.2f}".format(b,c,d))
        continue
    elif a==4:
        print("Masukkan Nilai Pertama : ",end="")
        b=float(input())
        print("Masukkan Nilai Kedua : ",end="")
        c=float(input())
        d=b/c
        print("Hasil pembagian antara {:.2f} dengan {:.2f} adalah {:.2f}".format(b,c,d))
        continue
    elif a==5:
        print("Terimakasih, telah menggunakan kalkulator Harry Pratama Yunus")
        break
    else :
        print("Input anda salah, silahkan coba lagi")
        continue